integration/tests/neon_evm/solana_native/test_scheduled_trx_balance.py

```diff
@@ -0,0 +1,561 @@
+import json
+import logging
+
+import eth_abi
+from eth_utils import abi
+
+from solders.pubkey import Pubkey
+
+from integration.tests.neon_evm.utils.contract import get_contract_bin
+from integration.tests.neon_evm.utils.ethereum import create_contract_address
+from utils.consts import LAMPORT_PER_SOL
+from utils.helpers import decode_function_signature
+from utils.scheduled_trx import ScheduledTransaction, CreateTreeAccMultipleData
+from utils.solana_logs_helper import parse_gas_used
+
+from utils.types import Contract
+
+LOG = logging.getLogger(__name__)
+
+LAMPORT_TO_INNER_SOL = 10**9
+OPERATOR_FEE_TO_NEON = 5_000
+
+
+def test_successful_single_trx_with_outer_deposit(
+    neon_user, evm_loader, operator_keypair, treasury_pool, basic_contract, neon_api_client, holder_acc
+):
+    # trx_status: successful
+    # user_balance: only outer deposit
+    # tree_acc: one schd trx in tree acc
+    iter_count_per_trx = 2
+    trx_count = 1
+
+    evm_loader.create_balance_account(neon_user.checksum_address, neon_user.solana_account, evm_loader.sol_chain_id)
+
+    operator_balance = evm_loader.get_operator_neon_balance(operator_keypair, evm_loader.sol_chain_id)
+    neon_user_balance_before = evm_loader.get_solana_balance(neon_user.solana_account.pubkey())
+    treasury_pool_balance = evm_loader.get_solana_balance(treasury_pool.account)
+
+    nonce = evm_loader.get_neon_nonce(neon_user.neon_address, evm_loader.sol_chain_id)
+    call_data = decode_function_signature("setNumber(uint256)", args=[10])
+
+    tx_0 = ScheduledTransaction(
+        neon_user.neon_address,
+        None,
+        nonce,
+        0,
+        target=basic_contract.eth_address,
+        value=0,
+        call_data=call_data,
+        chain_id=evm_loader.sol_chain_id,
+    )
+    tree_acc_data = CreateTreeAccMultipleData(nonce=nonce)
+    tree_acc_data.add_trx(tx_0, 0xFFFF, 0)
+    tree_acc = evm_loader.create_tree_account_multiple(neon_user, treasury_pool, tree_acc_data.data)
+
+    neon_user_balance_after_tree = evm_loader.get_solana_balance(neon_user.solana_account.pubkey())
+    delta_neon_user = neon_user_balance_before - neon_user_balance_after_tree
+    estimated_gas_fee = tx_0.DEFAULTS["gas_limit"] * tx_0.DEFAULTS["max_fee_per_gas"] / LAMPORT_TO_INNER_SOL
+    deposit_to_tree_acc = 10_000  # Where it comes from?
```

Comment:
from neon_user
it's like additional payment for tree acc deleting in the future (will be done by an operator)